cms/static/js/views/course_info_handout.js

```diff
@@ -50,10 +50,14 @@ define(['js/views/baseview', 'codemirror', 'common/js/components/views/feedback_
             },
 
             onSave: function(event) {
+                var dataToBeSaved = this.$codeMirror.getValue();
                 $('#handout_error').removeClass('is-shown');
                 $('.save-button').removeClass('is-disabled').attr('aria-disabled', false);
                 if ($('.CodeMirror-lines').find('.cm-error').length == 0) {
-                    this.model.set('data', this.$codeMirror.getValue());
+                    if (dataToBeSaved === '') {
```

Comment:
What about separating out this formatting logic from `onSave`? we may have a method which returns  '<ol></ol>' in case parameter passed `this.$codeMirror.getValue()` is empty. It would look like something `var dataToBeSaved = new_method(this.$codeMirror.getValue(), value_you_want_to_return_in_case_first_param_is_empty)`. Also you should consider changing `dataToBeSaved` to something that looks more relevant.